PalindromeLab/README.md

```diff
@@ -0,0 +1,26 @@
+<h1>Python Autograder Template with Twosum Demo</h1>
+
+<h2>Assessment Language</h2>
+Python
+
+<h2>Autograder Language</h2>
+Python
+
+<h2>Autograding Environment Packages</h2>
+Python3 and pip are needed for the autograding environment.
+
+<h2>Assessment Scenario</h2>
+In this lab, students are required to implement a function palindrome() which takes in an integer as input. The function checks whether or not the integer is a palindrome (the number is the same whether it is read backwards or forwards).
+
+The autograder also checks whether the function written by the student is time efficient.
+
+<h2>Handin Format</h2>
+palindrome.py
+
+<h2>Autograder.tar directory content</h2>
+<li>
```

Comment:
![Screenshot 2023-04-29 at 01 18 08](https://user-images.githubusercontent.com/9074856/235285010-f96329c2-5a1a-4d93-a604-7a67efde537b.png)

This isn't the right way to do a bullet list in Markdown. It should be something like:

```
- **Makefile:** Builds the lab
- **Src:** 
- etc.
```